### PR TITLE
[FIX] l10n_ee: update tax acct names to be unique

### PR DESCRIPTION
**Pull request by ethanrobv**
**Current behavior:**
Creating a new company and trying to switch to the l10n_ee accounting localization package will result in a validation error, preventing the installation.

**Expected behavior:**
The package should be installed and automatically activated for the active company.

**Steps to reproduce:**
0. Install the l10n_ee accounting module

1. Create a new company such that no fiscal localization package is active by default, then switch to this new company in the selection in the top right

2. In Accounting settings, select the Estonia fiscal localization package and save (this causes the exception)

**Cause of the issue:**
Certain tax accounts in the module do not have their names fully updated. They all were labeled as "0% ..." instead of their true percentages like all other tax accounts.

**Fix:**
Update the names of the offending tax accounts so that they are fully reflective of their true %.

Associated with https://github.com/odoo/enterprise/pull/56496

opw-3683538

### CODE REVIEW

File: addons/l10n_ee/data/template/account.tax-ee.csv [hunk 1]
```diff
@@ -203,13 +203,13 @@
 "","","","","","","","","","","tax","invoice","","","",""
 "","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","tax","refund","","","",""
-"l10n_ee_vat_in_0_kms_41_1","370","0% KMS §41¹","purchase","20.0","percent","0%","tax_group_vat_20","False","12","base","invoice","+1_1||+7_1","","","0% KMS §41¹"
+"l10n_ee_vat_in_0_kms_41_1","370","20% KMS §41¹","purchase","20.0","percent","0%","tax_group_vat_20","False","12","base","invoice","+1_1||+7_1","","","20% KMS §41¹"
```
Comment (yosa-odoo) [inline addons/l10n_ee/data/template/account.tax-ee.csv:206]: Maybe change the description so it is consistent with the invoice one ? 
`20% Special`

Same for the 22% one below

**Review by yosa-odoo — APPROVED** [review]
Thanks for the fix !

Tax names must be unique: https://github.com/odoo/odoo/blob/8a443222a116e8b9a94a4c8fc11798824abbc484/addons/account/models/account_tax.py#L200-L202

Makes sense to me (just the small remark that could be interesting to adjust)

**Review by aliyatastemirova — CHANGES_REQUESTED** [review]
Hi, thank you, my bad :slightly_smiling_face: 
- As far as I remember, there was a reason why these taxes were named with 0%. So maybe we could add the real percentage at the end, like `0% KMS §41¹ 20%`? The thing is, that next year the 20% taxes will most probably be removed. @dylankiss could you please remind me what is this tax about? Thank you :slightly_smiling_face: 
- The constraint is added in saas-16.4, so could you please retarget?

**Comment by dylankiss** [comment]
> * As far as I remember, there was a reason why these taxes were named with 0%. So maybe we could add the real percentage at the end, like `0% KMS §41¹ 20%`? The thing is, that next year the 20% taxes will most probably be removed. @dylankiss could you please remind me what is this tax about? Thank you 🙂

@aliyatastemirova This tax is used for specific products (e.g. metals) under the Estonian VAT Act (=KMS) §41¹.
I think I named it 0% back then, because the tax is added and then subtracted, so the net percentage on the invoice/bill is 0%. But maybe it makes sense to show the actual percentage. Not sure. For vendor bills, it doesn't matter I guess. But for customer invoices, it might be weird to show "22%", but see a 0.00 amount?

**Review by aliyatastemirova — APPROVED** [review]
olma is ok with this change

**Comment by aliyatastemirova** [comment]
@ethanrobv I think there is a test in l10n_ee_reports that is failing actually

**Comment by ethanrobv** [comment]
> @ethanrobv I think there is a test in l10n_ee_reports that is failing actually

https://runbot.odoo.com/runbot/build/58096562
Looks like a runbot issue

**Comment by aliyatastemirova** [comment]
https://runbot.odoo.com/runbot/build/58024421

**Comment by aliyatastemirova** [comment]
@jbw-odoo hi, could you please r+ this one too? thanks

**Pull request closed without merging**